src/utils/createTabNavigator.js

```diff
@@ -70,6 +72,30 @@ export default function createTabNavigator(TabView: React.ComponentType<*>) {
       return route.routeName;
     };
 
+    _getAccessibilityLabelText = route => {
```

Comment:
Can you change it to `({ route })` so that its consistent with other props and allows us to add more arguments in future?